libpldmresponder/fru.cpp

```diff
@@ -418,6 +418,7 @@ void FruImpl::removeIndividualFRU(const std::string& fruObjPath)
               << removeEntity.entity_container_id << " ]\n";
     associatedEntityMap.erase(fruObjPath); // sm00
 
+    deleteFruRecord(rsi);
     /* if (table
```

Comment:
Does this commented explanation makes any sense as we are deleting the fru record table entry now ?

Reply:
Removed the comment